src/styles/style.scss

```diff
@@ -0,0 +1,103 @@
+@import "utils/variables";
+@import "stars";
+
+body {
+  margin: 0;
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+.card {
```

Comment:
You should create a separate file for this BEM block (`card.scss`)
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/5fc69742-6ee9-4117-a497-7964e71c04d4)